testing/julia/APKBUILD

```diff
@@ -0,0 +1,169 @@
+# Contributor: Jakub Jirutka <jakub@jirutka.cz>
+# Maintainer: Jakub Jirutka <jakub@jirutka.cz>
+pkgname=julia
+pkgver=0.4.5
+# Keep in sync with deps/libuv.version.
+_libuv_ver=efb40768b7c7bd9f173a7868f74b92b1c5a61a0e
+# Keep in sync with deps/Versions.make.
+_rmathjulia_ver=0.1
+pkgrel=0
+pkgdesc="A high-level, high-performance dynamic language for technical computing"
+url="http://julialang.org"
+arch="x86 x86_64"
+license="MIT"
+depends="arpack
+	fftw>=3.3
+	gmp>=5.0
+	libgit2>=0.21
+	mpfr3
+	openblas
+	openlibm
+	openspecfun>=0.4
+	pcre2>=10.0
+	suitesparse>=4.1
+	"
+depends_dev="arpack-dev
+	dsfmt-dev
+	fftw-dev
+	gmp-dev
+	libgit2-dev
+	libunwind-dev
+	llvm-dev>=3.7
+	mpfr-dev
+	openblas-dev
+	openlibm-dev
+	openspecfun-dev
+	pcre2-dev
+	suitesparse-dev
+	utf8proc-dev>=1.3
+	"
+makedepends="$depends_dev
+	bash
+	cmake
+	gawk
```

Comment:
gawk is not needed, @tkelman fixed the install.sh issue https://github.com/JuliaLang/julia/pull/16784